charts/__tests__/ComparisonLineGenerator.test.ts

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env jest
+#! /usr/bin/env yarn jest
```

Comment:
@breck7 I think this way it uses `node_modules` rather than the globally installed `jest`. But let me know if there are possibly issues defining it like this

Reply:
This way with yarn is better, thanks!